pombola/feedback/templates/feedback/add.html

```diff
@@ -6,6 +6,15 @@
     'return_to_url':      return_to_url,
 {% endcomment %}
 
+{% if settings.GOOGLE_RECAPTCHA_SITE_KEY %}
+    <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+    <script>
+        function onSubmitFeedback() {
+            document.getElementById("add_feedback").submit();
+        }
```

Comment:
I'm a bit uncomfortable with this approach in this form, because whatever we call the callback, it will silently replace any other function in th global scope called onSubmitFeedback.

It looks like the function is optional when we don't want to run custom functionality before submitting - could you try removing the callback (also from the button) and seeing if it still works? https://developers.google.com/recaptcha/docs/display#render_param